HOWTO.md

```diff
@@ -264,4 +264,67 @@ Now add the new tool or language to the interface array according to what was ad
 
 And it's ready. Now horusec is inte.grated with the new tool and generating unified reports.
```

Comment:
How to add new tool using engine is deprecated please change docs

Reply:
Fixed